README.md

```diff
@@ -207,6 +207,20 @@ $ GHW_DISABLE_WARNINGS=1 ghwc memory
 memory (24GB physical, 24GB usable)
 ```
 
+You can disable warning programmatically using the `WithAlerter` option:
+
+```go
+
+import (
+	"io/ioutil"
+	"log"
+
+	"github.com/jaypipes/ghw"
+)
+
+mem, err := ghw.Memory(ghw.WithAlerter(log.New(ioutil.Discard, "", 0))
```

Comment:
we could make a singleton `ghw.NullAlerter` to make this user experience a bit better (and avoid the user needing to import `io/ioutil" and "log" packages themselves.

For instance, in `pkg/option/option.go` we could have:

```go
import (
    "io/ioutil"
     "log"
)

var (
    NullAlerter = &Alerter{
        log.New(ioutil.Discard, "", 0),
    }
)
```

and then the user would just need to do:

```go
mem, err := ghw.Memory(ghw.WithAlerter(ghw.NullAlerter))
```

We could make it even easier by making a `ghw.WithNullAlerter()` function, too, in `pkg/option/option.go`:

```go
// WithNullAlerter sets No-op alerting options for ghw
func WithNullAlerter(alerter Alerter) *Option {
	return &Option{
		Alerter: NullAlerter,
	}
}
```

after which all the user would need to do would be this:


```go
mem, err := ghw.Memory(ghw.WithNullAlerter())
```

Reply:
I like it, will update the PR soon(ish) to add this change.